bootstrap/sql/migrations/native/1.4.4/mysql/schemaChanges.sql

```diff
@@ -1 +1,13 @@
-ALTER TABLE user_entity ADD UNIQUE (name);
\ No newline at end of file
+ALTER TABLE user_entity ADD UNIQUE (name);
+
+-- Adding a new column 'taskAssigneesIds' to improve query performance by avoiding the use of JSON_EXTRACT in queries
+-- This column is generated to store the 'id' values extracted from the 'taskAssignees' JSON array, which helps in optimizing search and filtering operations.
+ALTER TABLE thread_entity
+ADD COLUMN taskAssigneesIds VARCHAR(1000) GENERATED ALWAYS AS (JSON_EXTRACT(taskAssignees, '$[*].id'));
```

Comment:
varchar(1000) size is enough? what if there are 50 users?

Reply:
no ..will not work for 50 max 20, will change to TEXT type

Reply:
not sure if TEXT is efficient in indexing